channel.py

```diff
@@ -16,95 +16,22 @@
 # Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
 
 import logging
-from random import random
 
 import gi
 from gi.repository import Gtk
 from gi.repository import Gdk
-from gi.repository import GObject
+from grepository import GObject
```

Comment:
I know very little about Gtk widgets, so i am just going to ask. Is this not a typo? If it's not, how come grepository instead of gi ? i.e. what specific feature make it preferable ?